client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -104,8 +105,10 @@ class SiteSettingsFormDiscussion extends Component {
 		const {
 			fields,
 			handleAutosavingToggle,
+			isJetpack,
```

Comment:
This one can also be removed once you clean up `isJetpackSite` from `JetpackModuleToggle`.